### DISCUSSION TITLE
models.CharField max_length attribute not working

### DISCUSSION THREAD

**Discussion by Bhuwan-web** [encode/django-rest-framework#8514] [Potential Issue] [answered]
# I have been in serious Problem due to this,
I have made an invoice_no filed max_length=12, and when i used random value it exceeded the value but it ran with no issue in local but no sooner as i went to production it throw an error, not a json api error as well, it straightforward sent me a server not found error and it took me hours to figureout what went wrong, Is it Django fault or i missed something??

**Comment by yyyyyyyan** [comment, marked as answer]
Hi! Since it's a problem with the model, Django REST framework probably has nothing to do with it. Can you post the error you're getting and the route/view that is resulting in the error?
